corehq/doctypemigrations/README.md

```diff
@@ -6,10 +6,36 @@
 If you're interested in running a migration someone else already wrote,
 see [Running the doctype migration](#run-the-doctype-migration) below.
 
+Do a full-text search for each doc_type you're migrating across all `map.js` files
+```bash
+$ grep -r --include=map.js CommCareUser
+```
+Are each of those views going to work properly after the doctype is migrated to the new database?
+
+Do any work you need to in order make sure that the doc_types in the apps you're migrating are decoupled
+from other ones:
+- You may need to break apart design documents so that:
+  - Any views that need to be in the new db will end up in the new db
+  - Any views that need to be in the current db will remain in the current db
+- You may need to have certain design documents show up on *both* the current and new dbs
+  - If there are functions that referenced these views wanting both,
+    you may need to rewrite those functions to check both dbs
+- etc.
+
+If the doctype you're migrating is read by pillowtop, you may need to reset the
+checkpoint after the merge, or in the case of elasticsearch pillows, trigger an
+index rebuild.
+We should figure out how to better deal with this issue next time it comes up.
```

Comment:
Yeah that's good to note. I think in the case of an elastic pillow, all you have to do is update the index name in your flip PR.